### PR TITLE
Issue 1

### PR DESCRIPTION
**Pull request by Tpirtle**
Issue 1 done

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 1]
```diff
@@ -27,6 +27,32 @@ public void LoadQuestions_RetrievesQuestionsFromFile()
             }
         }
 
+        //[TestMethod]
```
Comment (JerettLatimer) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs]: Good job. This is very similar to what I did when writing my test method.

Comment (breaman) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs]: Why did you change this from [TestMethod] to [DataTestMethod]? You aren't doing any data driven tests here.
